### PR TITLE
Fix some typos on vineyard doc

### PR DESCRIPTION
**Pull request by dashanji**
What do these changes do?
-------------------------

As titled

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -184,7 +184,7 @@ Vineyard is available as a `python package`_ and can be effortlessly installed u
 For comprehensive and up-to-date documentation, please visit https://v6d.io.
 
 If you wish to build vineyard from source, please consult the `Installation`_ guide. For
-instructions on building and running unittests locally, refer to the `Contributing`_ section.
+instructions on building and running unit tests locally, refer to the `Contributing`_ section.
```
Comment (sighingnow) [inline README.rst:187]: `unittests` is ok.

File: docs/notes/architecture.rst [hunk 2]
```diff
@@ -125,7 +125,7 @@ Distributed data sharing in big data tasks
 By examining the practices of big data tasks such as numeric computing, machine learning,
 and graph analysis, we have identified four key properties of the data involved:
 
-+ Distributed and each partitioned fragment usually fits into memory;
++ Distributed and each partitioned fragment usually fit into memory;
```
Comment (sighingnow) [inline docs/notes/architecture.rst:128]: fits.

File: docs/notes/architecture.rst [hunk 3]
```diff
@@ -184,7 +184,7 @@ rapidly along the processing pipeline.
 *NO* instant remote data accessing
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The partitions of a distributed data are stored distributedly in corresponding
+The partitions of distributed data are stored distributedly in corresponding
 Vineyard instances of the cluster. Only the client on the same machine can access
 the data partition. In order to access a remote partition, data migration APIs of
-Vineyard can be invoked to trigger the migration process, but not for instant accessing.
+Vineyard can be invoked to trigger the migration process, but not for instant access.
```
Comment (sighingnow) [inline docs/notes/architecture.rst:190]: accessing should be fine here.

File: docs/notes/key-concepts/objects.rst [hunk 4]
```diff
@@ -195,7 +195,7 @@ Distributed objects
 
 Vineyard is designed to store large objects across multiple nodes in a cluster, enabling
 user programs to seamlessly interact with these objects as a single entity. Data is
-sharded across multiple machines without replication.
+shared across multiple machines without replication.
```
Comment (sighingnow) [inline docs/notes/key-concepts/objects.rst:198]: sharded.

File: docs/tutorials/data-processing/using-objects-python.rst [hunk 5]
```diff
@@ -9,7 +9,7 @@ As discussed in :ref:`vineyard-objects`, each object in Vineyard consists of two
 2. The hierarchical metadata, which is shared across the entire Vineyard cluster
 
 Specifically, a ``Blob`` represents the unit where the data payload resides within a
-Vineyard instance. A blob object holds a segment of memory in the bulk store of the
+Vineyard instance. A blob object holds a segment of memory in the bulk storage of the
```
Comment (sighingnow) [inline docs/tutorials/data-processing/using-objects-python.rst:12]: store.

File: docs/tutorials/extending/define-datatypes-python.rst [hunk 6]
```diff
@@ -12,7 +12,7 @@ As discussed in :ref:`vineyard-objects`, each object in vineyard comprises two c
 2. The hierarchical meta data, which is shared across the entire vineyard cluster
 
 Specifically, a ``Blob`` represents the unit where the data payload resides in a vineyard
-instance. A blob object contains a segment of memory in the bulk store of the vineyard
+instance. A blob object contains a segment of memory in the bulk storage of the vineyard
 instance, allowing users to save their local buffer into a blob and later retrieve the
```
Comment (sighingnow) [inline docs/tutorials/extending/define-datatypes-python.rst:16]: store.

**Review by sighingnow — APPROVED** [review]

**Comment by dashanji** [comment]
All comments addressed. Thanks for correcting, PTAL.

**Review by sighingnow — APPROVED** [review]

**Pull request merged**